zipkin-lens/.npmrc

```diff
@@ -0,0 +1,2 @@
+# This is needed while dependencies rely on different versions of react 
+legacy-peer-deps=true
```

Comment:
unless you put this here, you have to remember not just for npm install, but also audit